docs/pages/getting-started.html

```diff
@@ -33,7 +33,7 @@ <h3 class="usa-heading">For developers</h3>
 
 <ol class="usa-content-list">
   <li>
-    <strong>HTML</strong> markup for the components are located in: 
+    <strong>HTML</strong> markup for the UI Components are located in: 
```

Comment:
As I thought! So this should be changed to the following:

"markup for the UI Components is located in:"